x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/crawler/domain_management/domain_management.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { EuiSpacer } from '@elastic/eui';
+
+import { Loading } from '../../../../../shared/loading';
+
+import { DeleteCrawlerDomainApiLogic } from '../../../../api/crawler/delete_crawler_domain_api_logic';
+import { GetCrawlerDomainsApiLogic } from '../../../../api/crawler/get_crawler_domains_api_logic';
+
+import { AddDomainFlyout } from './add_domain/add_domain_flyout';
+import { CrawlerStatusBanner } from './crawler_status_banner';
+import { DomainManagementLogic } from './domain_management_logic';
+import { DomainsPanel } from './domains_panel';
+import { EmptyStatePanel } from './empty_state_panel';
+
+export const SearchIndexDomainManagement: React.FC = () => {
+  DeleteCrawlerDomainApiLogic.mount();
```

Comment:
question: are these mounts still necessary with the connect you added to DomainManagementLogic?

Reply:
I couldn't get the logic working without specifically mounting it, which is contrary to my understanding of kea and the documentation but 🤷